src/snyk/common/views/treeNode.ts

```diff
@@ -7,22 +7,31 @@ export interface INodeIcon {
   ['dark']: string;
 }
 
-export const NODE_ICONS: { [key: string]: INodeIcon } = {
+type NODE_ICON_SEVERITY = 'critical' | 'high' | 'medium' | 'low';
+
+export const NODE_ICONS: { [key in NODE_ICON_SEVERITY]: INodeIcon } = {
   critical: {
+    light: path.join(__filename, '..', '..', '..', '..', '..', 'images', 'dark-critical-severity.svg'),
+    dark: path.join(__filename, '..', '..', '..', '..', '..', 'images', 'dark-critical-severity.svg'),
+  },
+  high: {
     light: path.join(__filename, '..', '..', '..', '..', '..', 'images', 'dark-high-severity.svg'),
     dark: path.join(__filename, '..', '..', '..', '..', '..', 'images', 'dark-high-severity.svg'),
   },
-  warning: {
+  medium: {
     light: path.join(__filename, '..', '..', '..', '..', '..', 'images', 'dark-medium-severity.svg'),
     dark: path.join(__filename, '..', '..', '..', '..', '..', 'images', 'dark-medium-severity.svg'),
   },
-  info: {
+  low: {
     light: path.join(__filename, '..', '..', '..', '..', '..', 'images', 'dark-low-severity.svg'),
     dark: path.join(__filename, '..', '..', '..', '..', '..', 'images', 'dark-low-severity.svg'),
   },
 };
 
-export type InternalType = { [key: string]: any };
+export type InternalType = {
+  nIssues?: number;
```

Comment:
@michelkaporin, is `nIssues` for number-of-issues?

Reply:
@pavel-snyk yes. I found it also not very readable (`issueCount` would be better), but didn't want to do renaming of the existing code not to balloon the PR.

Reply:
got it, thanks.
agree with you about do not make PR bigger :). Let's address it later.